testing/jormungandr-automation/resources/explorer/graphql/schema.graphql

```diff
@@ -419,17 +429,20 @@ type TransactionConnection {
   pageInfo: PageInfo!
 
   """A list of edges."""
-  edges: [TransactionEdge]
+  edges: [TransactionEdge!]!
```

Comment:
Why making TransactionEdge not nullable?

Reply:
same here